Sources/SmileID/Classes/Networking/Models/EnhancedKyc.swift

```diff
@@ -18,6 +18,7 @@ public struct EnhancedKycRequest: Codable {
     public let partnerId: String = SmileID.config.partnerId
     public let sourceSdk: String
     public let sourceSdkVersion: String
+    private var metadata: [Metadatum]
```

Comment:
On android we have this as an optional. Can we make it optional here as well and if not provided set it to `None`?